File: java/src/main/java/gov/doc/ntia/sigmf/examples/emitter/EmitterAnnotationExample.java
package gov.doc.ntia.sigmf.examples.emitter;

import gov.doc.ntia.sigmf.Annotation;
import gov.doc.ntia.sigmf.Global;
import gov.doc.ntia.sigmf.MetaDoc;
import gov.doc.ntia.sigmf.examples.Example;
import gov.doc.ntia.sigmf.examples.ExampleUtils;
import gov.doc.ntia.sigmf.ext.annotation.emitter.EmitterAnnotation;

import java.util.ArrayList;
import java.util.Calendar;
import java.util.List;

public class EmitterAnnotationExample implements Example {

    @Override
    public MetaDoc getExample() {
        MetaDoc metaDoc = new MetaDoc();
        Global global = ExampleUtils.getGlobal(Calendar.getInstance().getTime());
        metaDoc.setGlobal(global);
        List<Annotation> annotations = new ArrayList<>();
        EmitterAnnotation emitterAnnotation = new EmitterAnnotation();
        emitterAnnotation.setId("emiter_1");
        emitterAnnotation.setSampleStart(0l);
        emitterAnnotation.setSampleCount(1024l);
        emitterAnnotation.setLatitude(40.5);
        emitterAnnotation.setLongitude(-105.7);

        EmitterAnnotation emitterAnnotation2 = new EmitterAnnotation();
        emitterAnnotation2.setId("emiter_1");
        emitterAnnotation2.setSampleStart(1024l);
        emitterAnnotation2.setSampleCount(1024l);
        emitterAnnotation2.setLatitude(41.5);
        emitterAnnotation2.setLongitude(-105.3);
        annotations.add(emitterAnnotation);
        annotations.add(emitterAnnotation2);

        metaDoc.setAnnotations(annotations);
        return metaDoc;

    }
}

File: C#/sigmf/sigmf/examples/emitter/EmitterAnnotationExample.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace sigmf.examples.emitter
{
    public class EmitterAnnotationExample : Example
    {
        public override MetaDoc getExample() {
            MetaDoc metaDoc = new MetaDoc();
            Globals globals = ExampleUtils.getGlobals(getCurrentTime());
            metaDoc.setGlobal(globals);
            List<Annotation> annotations = new List<Annotation>();
            EmitterAnnotation emitterAnnotation = new EmitterAnnotation();
            emitterAnnotation.Id = "emitter_1";
            emitterAnnotation.SampleStart = 01;
            emitterAnnotation.SampleCount = 10241;
            emitterAnnotation.Latitude = 40.5;
            emitterAnnotation.Longitude = -105.7;

            EmitterAnnotation emitterAnnotation2 = new EmitterAnnotation();
            emitterAnnotation2.Id = "emitter_1";
            emitterAnnotation2.SampleStart = 10241;
            emitterAnnotation2.SampleCount = 10241;
            emitterAnnotation2.Latitude = 41.5;
            emitterAnnotation2.Longitude = -105.3;
            annotations.Add(emitterAnnotation);
            annotations.Add(emitterAnnotation2);

            metaDoc.setAnnotations(annotations);
            return metaDoc;
        }
        private DateTime getCurrentTime()
        {
            DateTime st = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
            TimeSpan t = (DateTime.Now.ToUniversalTime() - st);
            return st;
        }
    }
}
